c_cxx/README.md

```diff
@@ -10,59 +10,69 @@ This directory contains a few C/C++ sample applications for demoing onnxruntime
 # How to build
 
 ## Prerequisites
-1. Visual Studio 2015/2017/2019
-2. cmake(version >=3.13)
+1. Visual Studio 2019 or 2022
+2. cmake (version >= 3.24)
```

Comment:
should CMakeLists.txt be updated too?
https://github.com/microsoft/onnxruntime-inference-examples/blob/22fe8a46a2611602d28ea9d7fe8ced5fa5e37747/c_cxx/CMakeLists.txt#L4
there are some other CMakeLists.txt's specifying 3.13 too

Reply:
Went back to 3.13 as the CI doesn't have 3.24. Missed reverting this change.

Reply:
got it. we could install a more recent CMake on the CI machines in an earlier pipeline step. on the other hand, a lower version requirement may be more accommodating to users.

Reply:
yeah - I thought about the former and decided the latter was probably more important